client/components/list-end/docs/example.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ListEnd from 'components/list-end';
+
+export default function ListEndExample() {
+	return <ListEnd />;
+}
+ListEndExample.displayName = 'ListEndExample';
```

Comment:
Right right, but when defined such as in your case, the ListEndExample function will automatically have a `name` property of value `'ListEndExample'` — a property that AFAIK React and its devtools use, and a pattern that is common in our higher-order components:

```js
static displayName = `Localized(${ WrappedComponent.displayName || WrappedComponent.name || '' })`
```

Maybe there's a reason specific to devdocs examples to keep it, since it's stated in the docs. Otherwise, it shouldn't be needed for React.